test/local-fs.spec.ts

```diff
@@ -114,6 +115,14 @@ describe(`the local filesystem implementation`, () => {
                 .then(() => expect(fs.loadTextFile(fileName)).to.eventually.equals(newContent));
         });
 
+        it(`ignores events from ignored dir in watcher`, () => {
```

Comment:
I don't like the idea of adding specific error events to the API. 
I would rather starting with a single error event with 0 arguments. when a user gets this event she should drop all local caches and re-read the file system.
This affects all existing implementations and their tests